cmd/monaco/runner/runner.go

```diff
@@ -65,7 +67,12 @@ Examples:
 
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
 			log.PrepareLogging(fs, verbose, logSpy)
-			version.LogVersionAsInfo()
+
+			// log the version except for running the main command, help command and version command
+			if !strings.HasPrefix(cmd.Use, "monaco") && !strings.HasPrefix(cmd.Use, "help") && !strings.HasPrefix(cmd.Use, "version") {
```

Comment:
You could use cmd.Name() instead and check it directly instead of using HasPrefix
> // Name returns the command's name: the first word in the use line.